cmd/skaffold/app/cmd/flags.go

```diff
@@ -274,6 +275,7 @@ var flagRegistry = []Flag{
 		DefValue:      false,
 		FlagAddMethod: "BoolVar",
 		DefinedOn:     []string{"dev", "run"},
+		Deprecated:    "please use the `skaffold render` command instead",
```

Comment:
```suggestion
		Deprecated:    "please use the `skaffold render` command instead.",
```

Reply:
FYI did some digging on this because I wasn't sure what the recommended best practice is for punctuation in a deprecation warning. turns out there really isn't a style guide for deprecation warnings for client tooling, which seems a bit silly, but both `kubectl` and `go` use no punctuation for the actual error messages (which is an accepted convention), but **do** use punctuation for their calls to action, which this deprecation warning is. 

mostly just putting this here for anyone else who ends up following this thread since I didn't have a good justification for one way or another until now :)